weltyt.py

```diff
@@ -1,6 +1,191 @@
 import sqlite3
 
 
+def initializationDB(cursor):
+
+    with open("TYTDB.sql") as base:
+        for line in base:
+            cursor.execute(line)
+
+def show_my_resources(x):
+    for row in cur.execute("SELECT * FROM resources"):
+            print(row)
+
+    for rows in cur.execute("SELECT * FROM lands"):
+        print(rows)
+
+def buing_plants(x):
+    amount = input(
+                    "How many plants do you want to buy? Write only integer please. "
+                )
+            
+    try:
+        amount = int(amount)
+
+        money_amount = cur.execute(
+            'SELECT amount  FROM resources WHERE type = "Money" '
+        ).fetchone()[0]
+        plants_price = cur.execute(
+            'SELECT price FROM resources WHERE type = "Plants" '
+        ).fetchone()[0]
+        new_money_amount = money_amount - amount * plants_price
+        cur.execute(
+            f'UPDATE resources SET amount = {new_money_amount}  WHERE type = "Money"; '
+        )
+        plants_amount = cur.execute(
+            'SELECT amount  FROM resources WHERE type = "Plants" '
+        ).fetchone()[0]
+        new_plants_amount = plants_amount + amount
+        cur.execute(
+            f'UPDATE resources SET amount = {new_plants_amount}  WHERE type = "Plants"; '
+        )
+        print("Buying plants was done properly. ")
+    except ValueError:
+        
+        print("Invalid input, to buy any plant you have to write number as integer please. ")
+
+def buying_lands(x):
+    show_me_lands = input("If you want see lands offert, write yes please. ")
+    if show_me_lands == "yes":
+        print(lands_offert)
+
+    else:
+        print("Go directly to buying ")
+
+        
+        choose_class = input(
+            "What land class do you want to buy? Write only properly names of lands please. "
```

Comment:
I think that it is useful, because when you write input, you don't start writing already after dot, but you write input separated from previous statement. In this way it's more readable, I think so.

Reply:
You're right. We can add a newline or two here then - will be even more readable.